client/src/modules/CourseManagement/components/CourseModal/index.tsx

```diff
@@ -87,6 +90,18 @@ export function CourseModal(props: CourseModalProps) {
 
   const descriptionUrl = Form.useWatch('descriptionUrl', form);
 
+  const handleFieldsChange = (changedFields: any[], allFields: any[]) => {
+    const aliasField = changedFields.find(field => field.name[0] === 'alias');
+    const weAreCommunityUrlField = allFields.find(field => field.name[0] === 'wearecommunityUrl');
+
+    const isWeAreCommunityUrl = weAreCommunityUrlPattern.test(weAreCommunityUrlField.value);
+
+    if (aliasField && !isWeAreCommunityUrl) {
+      const defaultRSAppURL = `https://app.rs.school/registry/student?course=${aliasField.value}`;
+      form.setFieldsValue({ wearecommunityUrl: defaultRSAppURL });
```

Comment:
@aliakseitokarev this is incorrect: value should be initially set into the field, not appear on save or fields change. Antd has API for setting default values into fields, please consider using it

https://github.com/user-attachments/assets/2f792004-7b28-4994-ade8-c0f3267128da